capstone/src/main/java/com/step902020/capstone/OrganizationController.java

```diff
@@ -113,4 +113,15 @@ public RedirectView deleteOrganizationEvent(@RequestParam("event-id") String eve
     }
     return new RedirectView("manageevents.html", true);
   }
+
+  /**
+   * returns image with the same name as the user email from cloud storage
```

Comment:
That's excellent use of Javadoc -- to convey intent and let the user know what behavior to expect.